modules/node_modules/@ncigdc/utils/survivalplot.js

```diff
@@ -20,8 +20,8 @@ export const enoughData = (data: Object) => (
   data.results.length && data.results.every(r => r.donors.length >= MINIMUM_CASES)
 );
 
-async function fetchCurves(filters: ?Array<Object>): Promise<Object> {
```

Comment:
the default in the api is 1000 though right? is 25 even enough to make a useful chart?
we probably need to do something about that speed